### PR TITLE
react_redux-list-of-todos

### PR DESCRIPTION
**Pull request by svyatoslavPy**
[DEMO LINK](https://svyatoslavPy.github.io/react_redux-list-of-todos/)

### CODE REVIEW

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
Consider fixing previous comments and the test 
Screenshot 2024-07-17 at 16 57 17

File: src/App.tsx [hunk 1]
```diff
@@ -1,26 +1,64 @@
...
+            </div>
           </div>
         </div>
       </div>
-    </div>
 
-    <TodoModal />
-  </>
-);
+      <TodoModal showModal={showModal} onCloseModal={handleCloseModal} />
```
Comment (anastasiiavorobiova) [inline src/App.tsx:61]:
```suggestion
      showModal && <TodoModal onCloseModal={handleCloseModal} />
```

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
Consider fixing previous issues

File: src/components/TodoModal/TodoModal.tsx [hunk 2]
```diff
@@ -1,42 +1,24 @@
...
-            <strong className="has-text-danger">Planned</strong>
+        {loading && <Loader />}
 
-            {/* For completed */}
-            <strong className="has-text-success">Done</strong>
-            {' by '}
-            <a href="mailto:Sincere@april.biz">Leanne Graham</a>
-          </p>
-        </div>
+        {!loading && <ModalCard onCloseModal={onCloseModal} />}
```
Comment (etojeDenys) [inline src/components/TodoModal/TodoModal.tsx:20]: better to simplify it using the ternary operator

File: src/components/TodoList/TodoList.tsx [hunk 3]
```diff
@@ -1,222 +1,51 @@
...
+    return <h1 className="error-message">{error}</h1>;
+  }
 
-            <td className="is-vcentered is-expanded">
-              <p className="has-text-success">
-                illo est ratione doloremque quia maiores aut
-              </p>
-            </td>
+  return (
+    <>
```
Comment (etojeDenys) [inline src/components/TodoList/TodoList.tsx:34]: looks like a redundant fragment

https://react.dev/reference/react/Fragment#usage

**Review by etojeDenys — APPROVED** [review]
good job

**Pull request closed without merging**